health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -108,10 +108,30 @@ export const UICustomizations = {
-              }`;
-            }
+              const searchPlanConfig = async (body) => {
+                //assuming it will be success
+                const response = await Digit.CustomService.getResponse({
+                  url: "/plan-service/config/_search",
+                  useCache: false,
+                  method: "POST",
+                  userService: true,
+                  body
+                });
+                window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${String(parseInt(response?.PlanConfiguration[0]?.additionalDetails?.key || "0")+1)}&microplanId=${row.id}&campaignId=${
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)